packages/light-hoc/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
With babel 7, you can use `babel.config.js` (slightly easier IMHO working with JS, but since these stay simple - most of the time, small benefit)

Reply:
I prefer JS for one big benefit: you can add comments in your configs.